crates/sargon/src/signing/collector/extractor_of_instances_required_to_sign_transactions.rs

```diff
@@ -47,13 +47,19 @@ mod tests {
 
     #[test]
     fn preprocessor_init_fail() {
+        let intent_with_invalid_persona =
+            TransactionIntent::sample_entity_addresses_requiring_auth(
+                vec![],
+                vec![Persona::sample_mainnet().address],
+            );
+
         let result = ExtractorOfInstancesRequiredToSignTransactions::extract(
             &Profile::sample_other(),
-            vec![TransactionIntent::sample()],
+            vec![intent_with_invalid_persona],
             RoleKind::Primary,
         );
 
-        assert!(matches!(result, Err(CommonError::UnknownAccount)));
+        assert!(matches!(result, Err(CommonError::UnknownPersona)));
```

Comment:
Still invalid for unknown persona. We might change it in the future, but feels strange for a manifest to contain an unknown persona.